artview/core/common.py

```diff
@@ -229,25 +229,37 @@ def handleClick(button):
 # Option methods #
 ##################
 
+
+def float_or_none(text):
+    if text=="None":
+        return None
+    else:
+        return float(text)
+
```

Comment:
I actually want this function to raise an error in case of misuse, so that I can Handel this exception, for instance reset to previous value or standard value. I could add other spelling of `None`, but None is not to be a except value.